### PR TITLE
OHARA-1465 Support delete function in pipeline topic graph

### PR DESCRIPTION
**Pull request by eechih**
允許使用者刪除沒有任何 connection 的 topic

### CODE REVIEW

File: ohara-manager/client/src/components/pages/PipelinePage/connectors/Topic.js [hunk 1]
```diff
@@ -16,21 +16,17 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components';
+import { get, isEmpty, some, includes, omit } from 'lodash';
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/PipelinePage/connectors/Topic.js]: Hmm. I've been thinking about this lately. Let's remove all the lodash utility functions from `commonUtils` and use lodash directly in components that are needed these utils. What do you think about this?

File: ohara-manager/client/src/components/pages/PipelinePage/connectors/Controller/Controller.js [hunk 2]
```diff
@@ -17,16 +17,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import ReactTooltip from 'react-tooltip';
+import { noop, includes } from 'lodash';
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/PipelinePage/connectors/Controller/Controller.js]: Not super familiar with noop. Would you mind elaborate a bit why use this over primitive values like `String`, `Number` or `Null`? Thanks :)

File: ohara-manager/client/src/components/pages/PipelinePage/connectors/Topic.js (hunk 1 above)
Comment (konekoya) [reply ohara-manager/client/src/components/pages/PipelinePage/connectors/Topic.js]: Let's talk about this in person...

Comment (konekoya) [reply ohara-manager/client/src/components/pages/PipelinePage/connectors/Topic.js]: Alright, that's make this happend :)

File: ohara-manager/client/src/components/pages/PipelinePage/connectors/Controller/Controller.js (hunk 2 above)
Comment (eechih) [reply ohara-manager/client/src/components/pages/PipelinePage/connectors/Controller/Controller.js]: > Not super familiar with `noop`. Would you mind elaborate a bit why use this over primitive values like `String`, `Number` or `Null`? Thanks :)

const noop = () => undefined;

File: ohara-manager/client/src/components/pages/PipelinePage/connectors/Topic.js (hunk 1 above)
Comment (eechih) [reply ohara-manager/client/src/components/pages/PipelinePage/connectors/Topic.js]: 👍

File: ohara-manager/client/src/components/pages/PipelinePage/connectors/Controller/Controller.js (hunk 2 above)
Comment (konekoya) [reply ohara-manager/client/src/components/pages/PipelinePage/connectors/Controller/Controller.js]: Pretty cool. Maybe I'll need this in the future. Thanks for explaining all that.

**Comment by konekoya** [comment]
Thanks :)

**Pull request merged**